packages/models-library/src/models_library/services.py

```diff
@@ -37,14 +39,18 @@
 SERVICE_NETWORK_RE = r"^([a-zA-Z0-9_-]+)$"
 
 
-PROPERTY_TYPE_RE = r"^(number|integer|boolean|string|data:([^/\s,]+/[^/\s,]+|\[[^/\s,]+/[^/\s,]+(,[^/\s]+/[^/,\s]+)*\]))$"
-PROPERTY_KEY_RE = r"^[-_a-zA-Z0-9]+$"
+PROPERTY_TYPE_RE = r"^(number|integer|boolean|string|ref_contentSchema|data:([^/\s,]+/[^/\s,]+|\[[^/\s,]+/[^/\s,]+(,[^/\s]+/[^/,\s]+)*\]))$"
+PROPERTY_KEY_RE = r"^[-_a-zA-Z0-9]+$"  # TODO: PC->* it would be advisable to have this "variable friendly" (see VARIABLE_NAME_RE)
```

Comment:
you created a RE  in the basic_regex.py that you do not use?

Reply:
I am afraid that if I add this right now ... there will be more than one service that will start to fail. 
At this point I might add instead something that can convert that key into variable and back .